### ISSUE TITLE
Trailing space in list comprehension

### ISSUE THREAD

**Issue by jparoz** [commercialhaskell/hindent#357]
Running hindent on the following input:
```haskell
foo = [(x, y) | x <- [1..10], y <- [11..20], even x, odd y]
```
produces output:
```haskell
foo =
    [ (x, y)
    | x <- [1 .. 10] 
    , y <- [11 .. 20] 
    , even x 
    , odd y ]
```
There are trailing spaces on each line followed by a line beginning with `,`. Is this intended behaviour? I suspect they might be creeping in in the function `prefixedLined`, but I'm not really skilled enough to test or change anything.

**Comment by arianvp** [comment]
I got the same issue

**Comment by chrisdone** [comment]
That's not intended behaviour.

**Comment by sighingnow** [comment]
The issue seem already has been fixed in Hindent 5.2.2. Feel free to report if anyone could reproduce the same bug with the letest version of Hindent.
